presto-product-tests/README.md

```diff
@@ -143,6 +143,12 @@ presto-product-tests/bin/run_on_docker.sh <profile> -t com.facebook.presto.tests
 presto-product-tests/bin/run_on_docker.sh <profile> -t sql_tests.testcases.system.selectInformationSchemaTables
 ```
 
+Profile specific tests are tests which runs only for that particular profile. It confirms if the profile is same as what is claims.
```

Comment:
Keep lines to an 80 character width (except where you have a link or something that you can't break up). I see you changed line 116 to go over 80 characters so please break it up accordingly.